app/components/cases/subcases/new-subcase-form.js

```diff
@@ -223,6 +223,8 @@ export default class NewSubcaseForm extends Component {
           this.intl.t('warning-title')
         );
       }
+    } else {
+      yield this.agendaService.createInternalReview(this.subcase, null, privateComment);
```

Comment:
was wondering about this. subcase without agenda-activity.
It makes sense to not have the current "agendaitem.privateComment" yet.
But the process of their checklist can already happen with this new model so I already at an internalReview at this point.